pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --doctest-modules --doctest-glob='scrapinghub/*.py'
```

Comment:
@vshlapakov you either need `--doctest-modules` OR  `--doctest-glob='scrapinghub/*.py'` (I think the latter)